src/components/Button/styles.ts

```diff
@@ -28,5 +31,20 @@ export const StyledButton = styled('button', {
                 padding: '19px 82px',
             },
         },
+        iconSide: {
+            right: {
+                flexDirection: 'row-reverse',
+                '& svg ': {
+                    marginLeft: 10,
+                },
+            },
+            left: {
+                flexDirection: 'row',
+                '& svg ': {
+                    marginRight: 10,
+                },
+            },
+
+        }
```

Comment:
Tô começando a achar que os aninhamentos podem começar a fica difíceis de ler. Aqui você pode extrair partes que possuem aninhamentos "grandes", isolar em variáveis, só pra facilitar a leitura mesmo.

Reply:
não to sabendo como criar constantes aqui no stitches.. dei uma olhada na doc, não encontrei de que maneira poderia ser feito. Mas de toda maneira, percebi que o svg ali tem o mesmo estilo nos dois casos, então movi para a estilização geral.. assim não ficou tão aninhado

Reply:
Ah, não era específico da lib, era apenas fazer assim: 

```
const IconsSideStyles = {
            right: {
                flexDirection: 'row-reverse',
                '& svg ': {
                    marginLeft: 10,
                },
            },
            left: {
                flexDirection: 'row',
                '& svg ': {
                    marginRight: 10,
                },
            },

        }
```

e

```
iconSide: IconsSideStyles,
```

Reply:
prontinho!

Reply:
Bouah!